tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_create_new_alarm.py

```diff
@@ -95,4 +95,4 @@ def test_clock_create_new_alarm(self):
         self.alarm_alert.tap_stop_alarm()
 
         # Switch back to top level now that Clock app is gone
-        self.wait_for_condition(lambda m: self.apps.displayed_app.name == Homescreen.name)
+        self.wait_to_be_displayed()
```

Comment:
We're waiting on the homescreen to become visible here, so you need to do:
Homescreen(self.marionette).wait_to_be_displayed()